Archive/Wiedijk100Theorems/CubingACube.lean

```diff
@@ -585,7 +585,7 @@ theorem not_correct : ¬Correct cs := fun h =>
 
 /-- **Dissection of Cubes**: A cube cannot be cubed. -/
 theorem cannot_cube_a_cube :
-    ∀ {n : ℕ}, n ≥ 3 →                         -- In ℝ^n for n ≥ 3
+    ∀ {n : ℕ}, 3 ≤ n →                         -- In ℝ^n for n ≥ 3
```

Comment:
This no longer matches the comment.

Reply:
I think this is fine for a comment to be more mathematicall accurate than the corresponding Lean code?